0005-cmx2cdp-protocol/README.md

```diff
@@ -0,0 +1,207 @@
+---
+name: Communication protocol between CMX and CDP services
+shortname: 5/CMX2CDP
```

Comment:
Let's make sure we choose one of "CMX2CDP" or "cmx2cdp" and use it consistently.